hack/update-generated-code.sh

```diff
@@ -27,7 +27,7 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
 bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
-  github.com/kubernetes-csi/external-snapshotter/pkg/client github.com/kubernetes-csi/external-snapshotter/pkg/apis \
+  github.com/kubernetes-csi/external-snapshotter/client/v2 github.com/kubernetes-csi/external-snapshotter/client/v2/apis \
```

Comment:
Can you update hack/README.md?  There are steps on how to run update-generated-code.sh.  Now those steps need to be updated based on this change.  I think some extra steps can be removed now.

Reply:
@xing-yang  The script "/hack/update-crd.sh" doesn't work when i run this on root-directory,It is working when i run this in client directory.So is this ok to run this script from client directory?

Reply:
I think we should probably remove "v2" from line 30 of the script.  That's probably why v2 is actually in the final physical path.  However, since we've included v2 in the pkg name itself, we should not need it.

Reply:
@xing-yang yes agreed now when i run this script i got the output as: 

Generating deepcopy funcs
Generating clientset for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/clientset
Generating listers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/listers
Generating informers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/informers